src/main/java/baritone/process/MineProcess.java

```diff
@@ -452,4 +466,86 @@ public void mine(int quantity, BlockOptionalMetaLookup filter) {
             rescan(new ArrayList<>(), new CalculationContext(baritone));
         }
     }
+
+    public static boolean makeSphere(BlockPos pos, double radius, CalculationContext ctx) {
```

Comment:
This is extreme cringe. 

Literally everything about this function is cringe.

What is the use case for this? Ignore enteerman idk who that is.

Can't this be as simple as like:

```
for (int dx = -1; dx <= 1; dx++) {
    for (int dy = -1; dy <= 1; dy++) {
        for (int dz = -1; dz <= 1; dz++) {
            if (MovementHelper.isTransparent(pos.add(dx,dy,dz))) {
                return false;
            }
        }
    }
}
```

Reply:
leij's example is how I was doing spheres and enteerman or someone made fun of it :clown:

Reply:
enteerman is the real clown then?

this isn't intended to be a sphere, why would it be?

Reply:
# :joy_cat:

Reply:
yeah it shouldn't be a sphere

Reply:
i am bad at english and sorry if i didnt explain it well. It wasn't intended to be a sphere. just on some servers that blocks that are two blocks from a transparent block are all legit. I think hypixel does that in UHC so you can check that out. You can ignore me, I don't care. I sketched this out and this is how is should work: https://imgur.com/0O7BkWt

Reply:
I dont know why this method is large but just take the 6 blocks that are adjacent to the block, and take adjacents of the adjacents, to check if it is air.

Reply:
Or use ur method idk i dont play the game anymore, ask someone else's opinion on this

Reply:
for me @leijurv's method looks *way* better but it only checks the direct and diagonal neighbours. I'd rather use
 ```
 for (int dx = -radius; dx <= radius; dx++) {
     for (int dy = -radius; dy <= radius; dy++) {
         for (int dz = -radius; dz <= radius; dz++) {
             if (dx + dy + dz <= radius && MovementHelper.isTransparent(pos.add(dx,dy,dz))) {
                 return true;
             }
         }
     }
 }
 ```
to check all blocks that have a manhattan distance of at most `radius` (I think that is was @Enteerman wants to show with his image)